src/Microsoft.AspNet.Mvc.Xml/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "description": "Contains DataContractSerializer and XmlSerializer based input and output formatters",
+    "version": "6.0.0-*",
+    "compilationOptions": {
+        "warningsAsErrors": false
```

Comment:
I am not sure actually. I copied the content here from another project. Thought the details it has is useful anyways for this project too.